AraCorrelator/RayTraceCorrelator.cxx

```diff
@@ -297,11 +297,14 @@ std::vector<TGraph*> RayTraceCorrelator::GetCorrFunctions(
             throw std::invalid_argument(errorMessage);
         }
 
-        // get the correlation function
-        TGraph *grCorr = getCorrelationGraph_WFweight(
-            gr1_iter->second, 
-            gr2_iter->second
-            );
+        // It's a bit inefficient to reget the normalized traces every time,
+        // and not push this into the waveform getter. But shouldn't be a huge waste.
+        auto temp1 = getNormalisedGraphByRMS(gr1_iter->second);
```

Comment:
Another minutia comment, but could you rename these temp1 and temp2 variables so its more clear that these are traces (even though its in the comment above)? I got a bit turned around reading this the first time

Reply:
Done! See if it better now?